_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Custom Clock Processes
+nav: Custom Clock Processes
+modified_at: 2021-10-19 10:00:00
+tags: task-scheduling
+index: 2
+---
+
+With custom clock processes, you have the ability to specify custom schedule. You will also have other benefits like
+environment parity between development and production, or time precision in your task scheduling.
+
+## Definition of the Custom Clock
+
+The implementation of the custom clock will vary depending on the used language. However, defining the clock process
+is very simple and is done in the `Procfile`.
```

Comment:
As specified in our documentation (https://www.notion.so/scalingooriginal/Writing-Documentation-at-Scalingo-bb7985d6ea16455894417e36ede09cea#3b845d6a0b3f45f8808a90a3f650f65f), the words like "simple" shouldn't be used in documentation
```suggestion
 is done in the `Procfile`.
```